share/spack/gitlab/cloud_pipelines/stacks/rocm/spack.yaml

```diff
@@ -0,0 +1,155 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: true
+
+  config:
+    install_tree:
+      root: /home/software/spack
+      padded_length: 512
+      projections:
+        all: '{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'
+
+  definitions:
+    - default_specs:
+      - hip
+      - hipblas
+      - hipcub
+      - hipfft
+      - hipfort
+      - hipify-clang
+      # - hip-rocclr # Needs fix for install
+      - hipsolver
+      - hipsparse
+      # - hipsycl # Needs fixes for to use llvm-amdgpu clang
+      - hsa-rocr-dev
+      - mesa +llvm
+      - miopen-hip
+      - rocalution
+      - rocblas amdgpu_target=gfx1030
+      - rocfft amdgpu_target=gfx1030
+      - rocm-clang-ocl
+      - rocm-cmake
+      - rocm-dbgapi
+      - rocm-debug-agent
+      # - rocm-device-libs # built by llvm-amdgpu
```

Comment:
include rocm-device-libs too .. rocm-openmp-extras is dependent on rocmdevicelibs .it does not use the devicelibs built thru the llvm-amdgpu.